### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by dave-tucker**
Removes the Rust dispatchers as it won't ever be used.
This gets rid of `bpfd-common` and `bpfd-ebpf` crates.

### CODE REVIEW

File: bpfd/src/dispatcher_config.rs [hunk 1]
```diff
@@ -1,38 +1,34 @@
...
 // Copyright Authors of bpfd
 
-#![no_std]
-
 // XDP Defines
-pub const XDP_METADATA_SECTION: &str = "xdp_metadata";
-pub const XDP_DISPATCHER_VERSION: u32 = 1;
-pub const XDP_DISPATCHER_RETVAL: u32 = 31;
-pub const MAX_DISPATCHER_ACTIONS: usize = 10;
+// pub (crate) const XDP_METADATA_SECTION: &str = "xdp_metadata";
```
Comment (astoycos) [inline bpfd/src/dispatcher_config.rs:5]: Nit Remove these? (and the TC ones)

Comment (dave-tucker) [reply bpfd/src/dispatcher_config.rs:5]: They are there for a reason I promise 😆 
It's to jog my memory when I'm working on #319

Comment (astoycos) [reply bpfd/src/dispatcher_config.rs:5]: Sounds good!

**Review by astoycos — APPROVED** [review]
LGTM

**Pull request merged**